crates/test-utils/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
```

Comment:
Seems we have two manifeste in test-utils, what's the reason for this? Also the dump-katana bin is define twice

Reply:
This was copied by mistake, thanks for catching this, it has been removed via [this](https://github.com/kkrt-labs/kakarot-rpc/pull/553/commits/acd1869260b4d43af4642ce5624f8a0bdbb3f592) commit.